packages/flutter/lib/src/gestures/recognizer.dart

```diff
@@ -623,7 +632,8 @@ abstract class PrimaryPointerGestureRecognizer extends OneSequenceGestureRecogni
   ///
   /// Can be null to indicate that the gesture can drift for any distance.
   /// Defaults to 18 logical pixels.
```

Comment:
Should call out in the documentation that the default favors gestureSettings.touchSlop now, but other than that LGTM.

Reply:
`/// Defaults to gestureSettings.touchSlop with a fallback of 18 logical pixels.`?